content/docs/concepts/replication/disaster-recovery.md

```diff
@@ -44,7 +46,15 @@ b. Execute "reprotect" action on selected ReplicationGroup which will resume the
 ### Unplanned Migration to the target cluster/array
 This scenario is the typical choice when a site goes down: 
 
-a. Execute "failover" action on selected ReplicationGroup using the cluster name
+a. Scale down the application pods to zero replicas by editing the application manifest yaml file and changing the replicas count to 0.
```

Comment:
If the problem is that the Pod can not remount the volume, a `kubectl rollout restart deployment` is probably much better than setting replicas to 0.